syncer/checkpoint.go

```diff
@@ -103,7 +103,7 @@ func (b *binlogPoint) flush() {
 	b.flushedTI = b.ti
 }
 
-func (b *binlogPoint) rollback(schemaTracker *schema.Tracker, schema string) (isSchemaChanged bool) {
```

Comment:
I like the old name which is "schema.tracker" 😂 so if we use `filter.Table` we can keep it.

Besides, schema package only deals about upstream table structure related things.

Reply:
Is it reasonable in semantically that using `filter.Table`?🤔